update heading

mirnawong1 opened the pull request:
correct heading to match content and removed redundant content

## What are you changing in this pull request and why?
<!---
Describe your changes and why you're making them. If linked to an open
issue or a pull request on dbt Core, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
<!--
Uncomment if you're publishing docs for a prerelease version of dbt (delete if not applicable):
- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
-->
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

mirnawong1 committed:
Update develop-in-the-cloud.md

correct heading to match content and removed redundant content

runleonarun committed:
Merge branch 'current' into correct-heading

runleonarun commented on website/docs/docs/get-started/develop-in-the-cloud.md:
```diff
@@ -85,33 +85,23 @@ New to dbt? Check out our [Getting Started guide](/docs/get-started/getting-star
 The IDE uses developer credentials to connect to your data platform. These developer credentials should be specific to your user and they should *not* be super user credentials or the same credentials that you use for your production deployment of dbt.
 
-Follow the below steps to set up your developer credentials:
+Review the following steps to set up your developer credentials:
```
We could simplify this.
```suggestion
Set up your developer credentials:
```

runleonarun commented on website/docs/docs/get-started/develop-in-the-cloud.md:
```diff
@@ -85,33 +85,23 @@ New to dbt? Check out our [Getting Started guide](/docs/get-started/getting-star
 
 In order to start experiencing the great features of the Cloud IDE, you need to first set up a [dbt Cloud development environment](/docs/collaborate/environments/dbt-cloud-environments) and [developer credentials](#developer-credentials). If you're creating a new project, you will automatically configure this during the project setup. 
```
Addinf context to what this section contains and removing old link. 
```suggestion
In order to start experiencing the great features of the Cloud IDE, you need to first set up a [dbt Cloud development environment](/docs/collaborate/environments/dbt-cloud-environments). In the following steps, we outline how to set up developer credentials and access the IDE. If you're creating a new project, you will automatically configure this during the project setup. 
```

runleonarun commented on website/docs/docs/get-started/develop-in-the-cloud.md:
```diff
@@ -85,33 +85,23 @@ New to dbt? Check out our [Getting Started guide](/docs/get-started/getting-star
-
-### Access the Cloud IDE
-
 Now that you've set up your development environment and credentials, you should be able to access the Cloud IDE:
```
Is there a reason these steps are separate from setting up dev credentials? I'm not suggesting we combine them, just curious why they are separate. If we keep them separate, I made a suggestion in the into paragraph about making it clear what you will learn here. 

We could simplify this as well.
```suggestion
Access the Cloud IDE:
```

mirnawong1 replied:
good question and my initial thought was because they different steps on diff pages, i wanted to make it clear to the user that they have to do xx things on this page and xx things on that page. however, your question makes me believe that if someone has to always create their dev credentials and then go to **develop** to access the ide, maybe i should combine the steps. I've done that any to see how it flows!

runleonarun reviewed (changes requested):
Just a few small suggestions @mirnawong1! Otherwise awesome changes to simplify this section.

runleonarun commented:
@mirnawong1  Grace asked abut this page [here](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1676054744014159) and it makes me think we should:
* Include a redirect for this page: https://docs.getdbt.com/docs/getting-started/develop-in-the-cloud%23setting-up-developer-credentials
* How can we help people linking directly to the subheading now that we've removed it? Like this: `https://docs.getdbt.com/docs/get-started/develop-in-the-cloud#developer-credentials`

mirnawong1 committed:
Update website/docs/docs/get-started/develop-in-the-cloud.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/get-started/develop-in-the-cloud.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into correct-heading

mirnawong1 committed:
Update develop-in-the-cloud.md

making credentials and access ide steps as one set of instructions since they would always f

mirnawong1 committed:
Update website/docs/docs/get-started/develop-in-the-cloud.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

mirnawong1 committed:
Update develop-in-the-cloud.md

Commit:
fix redirects and flow

mirnawong1 commented:
> @mirnawong1 Grace asked abut this page [here](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1676054744014159) and it makes me think we should:
> 
> * Include a redirect for this page: https://docs.getdbt.com/docs/getting-started/develop-in-the-cloud%23setting-up-developer-credentials
> * How can we help people linking directly to the subheading now that we've removed it? Like this: `https://docs.getdbt.com/docs/get-started/develop-in-the-cloud#developer-credentials`

thank you @runleonarun and I've added redirects for this and the previous subheaders. that's a great question - i will ask the  front end team to see if there is another way to other than redirects.

mirnawong1 committed:
Update develop-in-the-cloud.md

runleonarun commented on website/docs/docs/get-started/develop-in-the-cloud.md:
```diff
@@ -75,43 +75,33 @@ The Cloud IDE needs explicit action to save your changes. There are three ways y
-2. Click **Develop** at the top of the page to take you to the IDE.
-3. Initialize your project and familiarize yourself with the IDE and its delightful [features](#cloud-ide-features).
-4. Nice job, you're ready to start developing and [building models](https://docs.getdbt.com/docs/get-started/getting-started/building-your-first-project/build-your-first-models) 🎉! 
+Nice job, you're ready to start developing and [building models](https://docs.getdbt.com/docs/get-started/getting-started/building-your-first-project/build-your-first-models) 🎉! 
```
Make sure to use relative links so the fails when they are broken.
```suggestion
Nice job, you're ready to start developing and [building models](/docs/get-started/getting-started/building-your-first-project/build-your-first-models) 🎉! 
```

runleonarun replied:
I do think this flows better with the steps combined!

I wonder if it's not the best UX to send them into the middle of the Getting Started guide at this point? We have the next section on this page called [Build, compile and run...](https://deploy-preview-2834--docs-getdbt-com.netlify.app/docs/get-started/develop-in-the-cloud#build-compile-and-run-projects). Should we send them there? I wonder if this is pointing to a bigger problem of does the "Build, compile, run..." section belong on this page, or should it be a separate page (we don't need to solve this here - we could open an issue for this)? Also where did they come from to get here? Are they in the Getting started section? 

I know @nghi-ly is doing some work to tease apart the sample data/warehouse vs using your own data /warehouse, so maybe this will be a page we need OR this content can be moved from the "Build, compile, run..." section to the `getting-started/building-your-first-project/build-your-first-models` page? If you come up with any ideas or clear decisions, you can capture them in a new issue. If it's best left as-is, then that's fine too!

mirnawong1 replied:
gotcha! i first linked it to the 'build models' months ago because i had no idea they were linked to the guides and thought they were stand-alone pages 🙈 but since they're linked to the guides, I've removed the link as it doesn't make since the next section touches upon that. thank you @runleonarun !!

I've made those changes and think by removing that, it separates this doc from the guides, which allows the user to either use the ide for their own data or to follow the guides, if they want.

runleonarun reviewed (commented):
@mirnawong1 I have one last comment /thought about where this Devlop in the Cloud page fits in with other workflows. Feel free to move your thoughts into an issue, maybe labeled with "getting started" 

Once you do that, I think we can ship this work!

mirnawong1 committed:
Update website/docs/docs/get-started/develop-in-the-cloud.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into correct-heading

Commit:
updates

Commit:
Merge branch 'correct-heading' of https://github.com/dbt-labs/docs.getdbt.com into correct-heading

mirnawong1 committed:
Merge branch 'current' into correct-heading

mirnawong1 committed:
Merge branch 'current' into correct-heading

mirnawong1 committed:
Update _redirects

mirnawong1 committed:
Merge branch 'current' into correct-heading

mirnawong1 committed:
Merge branch 'current' into correct-heading

runleonarun committed:
Merge branch 'current' into correct-heading

runleonarun reviewed (approved):
:shipit:

runleonarun committed:
Merge branch 'current' into correct-heading
